test/utils/pickers/describeValue/testPickerActionBar.tsx

```diff
@@ -156,15 +156,15 @@ export const testPickerActionBar: DescribeValueTestSuite<any, 'picker'> = (
           onClose,
           open: true,
           defaultValue: values[0],
-          slotProps: { actionBar: { actions: ['accept'] } },
+          slotProps: { actionBar: { actions: ['accept', 'nextOrAccept'] } },
           closeOnSelect: false,
         });
 
         // Change the value (already tested)
         setNewValue(values[0], { isOpened: true, selectSection, pressKey });
 
         // Accept the modifications
-        fireEvent.click(screen.getByText(/ok/i));
+        fireEvent.click(screen.getAllByRole('button', { name: 'OK' })[0]);
```

Comment:
Are there multiple `OK` buttons in the action bar at any time? 🤔

Reply:
Yep, when you have `nextOrAccept` and `accept` actions and you are on the last step.
In which case we can click either of them, the result will be the same.